pages/membership/general.tsx

```diff
@@ -1,5 +1,5 @@
 import GalleryRoute from 'scenes/_Router/GalleryRoute';
-import PartnerMembershipMintPage from 'scenes/MembershipMintPage/PartnerMembershipMintPage';
+import GeneralMembershipMintPage from 'scenes/MembershipMintPage/GeneralMembershipMintPage';
```

Comment:
renamed `PartnerMembershipMintPage` to `GeneralMembershipMintPage` to reflect route